code/modules/mob/living/carbon/human/death.dm

```diff
@@ -8,15 +8,27 @@
 		if(GIB_TYPE_ROBOTIC)
 			new /obj/effect/temp_visual/gib_animation(loc, "gibbed-r")
 
+/mob/living/carbon/human/dust(just_ash, drop_items, force)
+	death(TRUE)
+
+	if(drop_items)
+		unequip_everything()
+
+	if(buckled)
+		buckled.unbuckle_mob(src, force = TRUE)
+
+	dust_animation()
+	QDEL_IN(src, 20)
+
 /mob/living/carbon/human/dust_animation()
-	if(!dna)
-		new /obj/effect/temp_visual/dust_animation(loc, "dust-h")
-		return
-	switch(dna.species.species_gibs)
-		if(GIB_TYPE_HUMAN)
-			new /obj/effect/temp_visual/dust_animation(loc, "dust-h")
-		if(GIB_TYPE_ROBOTIC)
-			new /obj/effect/temp_visual/dust_animation(loc, "dust-r")
+	// Animate them being dusted out of existence
+	var/obj/effect/dusting_anim/dust_effect = new(loc, src)
```

Comment:
render_target should be the same as render_source in the filter so you should probably make it send REF(src) instead of src itself

Reply:
52ce5bf